### PR TITLE
Created TimeDiff

### PR DESCRIPTION
**Pull request by elivina997**

### CODE REVIEW

File: app/src/main/java/com/univer/covid19help/MainFragment.kt [hunk 1]
```diff
@@ -25,12 +25,11 @@ import java.util.*
 class MainFragment : Fragment() {
 
     companion object {
-        val HOUR: Long = 1000*60*60
-        val MINUTE: Long = 1000*60
+        const val HOUR: Long = 1000*60*60
```
Comment (SergeyPjatkin) [inline app/src/main/java/com/univer/covid19help/MainFragment.kt:28]: move HOUR and MINUTES to TimeDiff.kt file

File: app/src/main/java/com/univer/covid19help/TimeDiff.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    enum class Diff { TOO_EARLY, LESS_THAN_HOUR, LESS_THEN_30_MINUTES, LESS_THEN_15_MINUTES, LESS_THEN_5_MINUTES, NOW, TOO_LATE, NONE }
+
+    private var alreadyLessThanHour = false
+    private var alreadyLessThan30Min = false
+    private var alreadyLessThan15Min = false
+    private var alreadyLessThan5Min = false
+    private var alreadyWasNow = false
+
+    fun resetTimeDiff() {
+        alreadyLessThanHour = false
```
Comment (SergeyPjatkin) [inline app/src/main/java/com/univer/covid19help/TimeDiff.kt:16]: Consider to rename to simple reset()

File: app/src/main/java/com/univer/covid19help/TimeDiff.kt [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        alreadyLessThan30Min = false
+        alreadyLessThan15Min = false
+        alreadyLessThan5Min = false
+        alreadyWasNow = false
+    }
+
+    fun getDiff(current: Date, future: Date): Diff {
```
Comment (SergeyPjatkin) [inline app/src/main/java/com/univer/covid19help/TimeDiff.kt:23]: Kotlin style do not uses get/set. COnsider to rename just diff()

File: app/src/main/java/com/univer/covid19help/MainFragment.kt (hunk 1 above)
Comment (elivina997) [reply app/src/main/java/com/univer/covid19help/MainFragment.kt]: Done

File: app/src/main/java/com/univer/covid19help/TimeDiff.kt (hunk 2 above)
Comment (elivina997) [reply app/src/main/java/com/univer/covid19help/TimeDiff.kt:20]: Done

File: app/src/main/java/com/univer/covid19help/TimeDiff.kt (hunk 3 above)
Comment (elivina997) [reply app/src/main/java/com/univer/covid19help/TimeDiff.kt]: Done

**Review by SergeyPjatkin — APPROVED** [review]

**Pull request merged**